data/duke.txt

```diff
@@ -0,0 +1,4 @@
+T | 1 | read book
+D | 0 | return book | June 6th
+E | 0 | project meeting | Aug 6th 2-4pm
+T | 1 | join sports club
```

Comment:
Its better to have such files be part of gitignore, as there is no need to track the revision history for such files